packages/libs/react-ui/src/components/TreeList/TreeList.css.ts

```diff
@@ -0,0 +1,63 @@
+import { sprinkles } from '@theme/sprinkles.css';
+import { style, styleVariants } from '@vanilla-extract/css';
+
+export const treeWrapperClass = style([
+  sprinkles({
+    color: '$foreground',
+  }),
+]);
+
+export const treeTitleClassWrapper = style([
+  sprinkles({
+    display: 'flex',
+  }),
+]);
+
+export const treeTitleClass = style([
+  sprinkles({
+    marginLeft: '$1',
+  }),
+]);
+export const treeTitleVariant = styleVariants({
+  isParent: [
+    sprinkles({
+      fontSize: '$base',
+    }),
+  ],
+  isChild: [
+    sprinkles({
+      fontSize: '$sm',
+    }),
+  ],
+});
+
+export const treeToggleClass = style([
+  {
+    border: 'none',
+    background: 'none',
+    outline: 'none',
+  },
+]);
+export const treeToggleVariant = styleVariants({
+  opened: [{ transform: 'rotate(0deg)' }],
+  closed: [{ transform: 'rotate(-90deg)' }],
+});
+
+export const treeBranchWrapperClass = style([
+  {
+    transition: 'all 0.2s ease',
+    overflow: 'hidden',
+  },
+]);
+export const treeBranchWrapperVariant = styleVariants({
+  isParent: [
+    {
+      marginLeft: '0',
```

Comment:
can use a sprinkle here

Reply:
nope, it will throw error saying "0" is not recognized as a value

Reply:
I just tried it out and it is a bit strange - it works but only when you pass the 0 as a number instead of a string
```suggestion
      marginLeft: 0,
```